### DISCUSSION TITLE
[QUESTION] Is there a way to use the setup.sh to install an older version?

### DISCUSSION THREAD

**Discussion by jd-apprentice** [coreemu/core#803] [General]
**Describe the bug**

It's not a bug, I'm trying to use the `core-gui-legacy` command but as I saw in the latest version it was removed so I was thinking about installing the 7.5.2 but no case, I'm just able to install the software via the `setup.sh + inv install` method. Is there any guide to install older versions?

**Expected behavior**

Use `core-gui-legacy`

**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 22.04] VM
 - CORE Version [e.g. 7.5.2]

**Comment by bharnden** [comment]
For most recent older version, the `docs` directory will have install instructions and should be mostly similar.

The `core-gui-legacy` was just a name as part of a transition for removing it. Prior it would just have been `core-gui`.
